def raise_exception():
    n = 10 / 0
    return n